features/waarden.feature

```diff
@@ -6,83 +6,147 @@ Feature: leveren van waarden bij een WOZ-object
-    - cassatie ingesteld (30)
+    - bezwaaringediend (10)
+    - beroepaangetekend (20)
+    - hogerberoepaangetekend (23)
+    - cassatieingesteld (30)
 
 
   Scenario: meerdere jaren met zelfde eigenaar en zonder bezwaar of beroep
-    Gegeven WOZ-object met objectnummer "051800823525" heeft de volgende beschikkingen voor eigenaar R. Geels   :
-      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
-      | 01-01-2019 | 431.000 euro | R. Geels       | 01-01-2020   | 11-02-2020        |
-      | 01-01-2018 | 390.000 euro | R. Geels       | 01-01-2019   | 12-02-2019        |
-      | 01-01-2017 | 340.000 euro | R. Geels       | 01-01-2018   | 13-02-2018        |
-      | 01-01-2016 | 340.000 euro | R. Geels       | 01-01-2017   | 14-02-2017        |
-      | 01-01-2015 | 310.000 euro | R. Geels       | 01-01-2016   | 16-02-2016        |
-      | 01-01-2014 | 308.000 euro | R. Geels       | 01-01-2015   | 17-02-2015        |
+    Gegeven WOZ object met identificatie "051800823525" heeft de volgende beschikkingen:
+    | Peildatum  | WOZ-waarde   | Ingangsdatum | Datum beschikking |
+    | 01-01-2019 | 431.000 euro | 01-01-2020   | 11-02-2020        |
+    | 01-01-2018 | 390.000 euro | 01-01-2019   | 12-02-2019        |
+    | 01-01-2017 | 340.000 euro | 01-01-2018   | 13-02-2018        |
+    | 01-01-2016 | 340.000 euro | 01-01-2017   | 14-02-2017        |
+    | 01-01-2015 | 310.000 euro | 01-01-2016   | 16-02-2016        |
+    | 01-01-2014 | 308.000 euro | 01-01-2015   | 17-02-2015        |
     Als het WOZ object wordt opgevraagd met /wozobjecten/051800823525
-    Dan zijn de waarden:
+    Dan levert de API de volgende waarden:
     """
-    "waarden": [
+    [
       {
         "vastgesteldeWaarde": 431000,
-        "waardepeildatum": "2019-01-01"
+        "waardepeildatum": "2019-01-01",
+        "ingangsdatum": "2020-01-01",
+        "beschikkingsStatussen": [ "beschikking_genomen" ]
+      },{
+        "vastgesteldeWaarde": 390000,
+        "waardepeildatum": "2018-01-01",
+        "ingangsdatum": "2019-01-01",
+        "beschikkingsStatussen": [ "beschikking_genomen" ]
       },
+        "vastgesteldeWaarde": 340000,
+        "waardepeildatum": "2017-01-01",
+        "ingangsdatum": "2018-01-01",
+        "beschikkingsStatussen": [ "beschikking_genomen" ]
+      },{
+        "vastgesteldeWaarde": 340000,
+        "waardepeildatum": "2016-01-01",
+        "ingangsdatum": "2017-01-01",
+        "beschikkingsStatussen": [ "beschikking_genomen" ]
+      },{
+        "vastgesteldeWaarde": 310000,
+        "waardepeildatum": "2015-01-01",
+        "ingangsdatum": "2016-01-01",
+        "beschikkingsStatussen": [ "beschikking_genomen" ]
+      },{
+        "vastgesteldeWaarde": 308000,
+        "waardepeildatum": "2014-01-01",
+        "ingangsdatum": "2015-01-01",
+        "beschikkingsStatussen": [ "beschikking_genomen" ]
+      }
+    ]
+    """
+    En zijn de daaruit bepaalde relevante de waarden:
```

Comment:
Hier is een 'de' teveel. 'En zijn de daaruit bepaalde relevante **de** waarden' => 'En zijn de daaruit bepaalde relevante waarden'